boot.sh

```diff
@@ -1,30 +1,37 @@
+#!/bin/bash
 # Boot up github shell access for Windows WSL/Ubuntu.
 
-#!/bin/bash
+if [ "$#" -lt 1 ]
```

Comment:
@ajaygtm71 , we want to make the location parameter optional here, and not mandatory. Probably behind a flag such as `-l <location>` or `--location <location>`.